.github/workflows/ci.yaml

```diff
@@ -0,0 +1,73 @@
+name: EC2 Instance Selector CI and Release
```

Comment:
Can we have this consistent to the repo name as: ``Amazon EC2 Instance Selector CI and Release``

Reply:
I shortened it because GH actions page cuts off the name of the workflow. But I think this still maintains consistency since the binary and the tool is named EC2 Instance Selector, the repo is Amazon EC2 Instance Selector.